docs: update README with a content optimizations for SEO (#784)
exadel-inc/esl: README.md

[![License](https://img.shields.io/badge/license-MIT-green.svg?style=for-the-badge)](./README.md)

Exadel Smart Library (**ESL**) is a web components based UI components library that gives you a set of 
**lightweight** and **flexible** custom elements to easily create basic UX modules and make your websites super fast.
ESL web UI components library is compatible with any framework which makes ESL ideal for projects regardless of their tech stack

<p align="center">
<a href="https://esl-ui.com"><img src="./docs/images/welcome-btn.png" alt="Visit our UI component library website with examples" width="409" height="75" title="Click to visit our UI component library website with examples"/></a>
</p>

<p align="center" >★ <b>Check out our UI component library and support it with a star</b> ★</p>

## Overview

Exadel Smart Library, or ESL for short, is a free web UI components library that reduces routine tasks 
and lets you efficiently create websites with multiple web components at hand. 
While looking for the best UI component library, keep in mind that it should be compatible with other libraries 
and frameworks. And ESL is!

> The UI component library allows you to access the toolkit full of reusable components. 

Reviewer: ```suggestion
The UI component library allows you to access the toolkit with plenty of reusable components. 
```

Author: Updated

You can utilize these web components to complement projects or build your infrastructure from the ground 
up using ESL alone. Our web UI components library is accurate and you can easily work around styles 
to make your projects consistent.

> ESL based just on Web API and ECMAScript 6 features which let our web UI components library 

Reviewer: ```suggestion
ESL is based just on Web API and ECMAScript 6 features which let our web UI components library 
```

Author: Updated

> work well across all modern browsers such as Firefox, Chrome, Safari, Opera, or Edge. ESL can 

Reviewer: ```suggestion
to work well across all modern browsers such as Firefox, Chrome, Safari, Opera, or Edge. ESL can 
```

also work on IE11 when the whole list of polyfills is provided (see [Browser support & Polyfills](https://github.com/exadel-inc/esl/blob/HEAD/docs/BROWSER_SUPPORT.md) for more details). 

## Library Structure
### Base Components

> With our UI component library, you get flexible syntax sugar meant for building over basic components and controlling the life cycle of those components.

Reviewer: ```suggestion
With our UI component library, you are getting flexible syntax sugar meant for building over basic components and controlling the life cycle of those components.
```

[![build](https://img.shields.io/github/workflow/status/exadel-inc/esl/validate/main?style=for-the-badge)](https://github.com/exadel-inc/esl/actions/workflows/validate.yml)
[![License](https://img.shields.io/badge/license-MIT-green.svg?style=for-the-badge)](./README.md)

> Exadel Smart Library (**ESL**) is an open-source **lightweight** and **flexible** UI component library based on web components.

Reviewer 2: I suggest write "UI components (plural form) library" throughout the document.

Author: Have confirmation from the SEO team that's the form of keyword important here and unfortunately
"UI component library" is a popular keyword but the "UI components library" is not (

> Build UX modules with custom components provided by our library and make sites work faster. 

Reviewer: ```suggestion
Build UX modules with custom components provided by our library to make sites work faster. 
```

ESL web UI components library is compatible with any framework which makes ESL ideal for projects regardless of 
their tech stack.

<p align="center">
<a href="https://esl-ui.com"><img src="./docs/images/welcome-btn.png" alt="Visit our UI component library website with examples" width="409" height="75" title="Click to visit our UI component library website with examples"/></a>
</p>

<p align="center" >★ <b>Check out our UI component library and support it with a star</b> ★</p>

- [Overview](#overview)
- [Library Structure](#library-structure)
- [Installation Guide](#installation-guide)
- [Roadmap](#roadmap)
- [Development Information for Contributors](#development-information-for-contributors)

## Overview

Exadel Smart Library, or ESL for short, is a free web UI components library that reduces routine tasks and lets 
you efficiently create websites with multiple web components at hand. 
While looking for the best UI component library, keep in mind that it should be compatible with other libraries 

> and frameworks. And ESL is!

Reviewer: ```suggestion
and frameworks. And ESL does!
```

Author: It looks like the original one is more proper here.

- ##### [ESL Animate](https://github.com/exadel-inc/esl/blob/HEAD/src/modules/esl-animate/README.md) (beta)
- ##### [ESL Popup](https://github.com/exadel-inc/esl/blob/HEAD/src/modules/esl-popup/README.md) (beta)

- ##### [ESL Select](https://github.com/exadel-inc/esl/blob/HEAD/src/modules/esl-forms/esl-select/README.md)
- ##### [ESL Select List](https://github.com/exadel-inc/esl/blob/HEAD/src/modules/esl-forms/esl-select-list/README.md)

### Utilities

> Our web UI components library uses utilities to build components with simpler syntax sugar and a big set of helpers.

Author: ```suggestion
Our web UI components library uses utilities to build components using simplifying syntax sugar and a big set of helpers.
```
